web_ui/frontend/app/(login)/login/page.tsx

```diff
@@ -78,86 +76,111 @@ export default function Home() {
+                <Grow in={error !== undefined}>
+                    <Typography
+                        textAlign={"center"}
+                        variant={"subtitle2"}
+                        color={"error.main"}
+                        mb={1}
+                    >
+                        {error}
+                    </Typography>
+                </Grow>
+                <LoadingButton
+                    variant="outlined"
+                    sx={{margin: "auto"}}
+                    color={"primary"}
+                    type={"submit"}
+                    loading={loading}
+                >
+                    <span>Confirm</span>
+                </LoadingButton>
+            </Box>
+        </form>
+    )
+
+    if(enabledServers && enabledServers.includes("registry")) {
+        return (
+            <Box display={"flex"} flexDirection={"column"} justifyContent={"center"}>
+                <Box m={"auto"}>
+                    <Button size={"small"} variant={"text"} onClick={() => setToggled(!toggled)}>
+                        Server Admin Login
+                    </Button>
+                </Box>
+                <Collapse in={toggled}>
+                    {LoginComponent}
+                </Collapse>
+            </Box>
+        )
+    }
+
+    return LoginComponent
+}
+
+export default function Home() {
+
+    const {data: enabledServers} = useSWR<ServerType[]>("getEnabledServers", getEnabledServers)
+
     return (
         <>
-            <Box m={"auto"} mt={12}  display={"flex"} flexDirection={"column"}>
+            <Box m={"auto"} mt={"20vh"}  display={"flex"} flexDirection={"column"}>
                 <Box>
                     <Typography textAlign={"center"} variant={"h3"} component={"h3"}>
                         Login
                     </Typography>
+                    <Box color={"grey"} mt={1} mb={2}>
+                        <Typography textAlign={"center"} variant={"h6"} component={"p"}>
+                            Administer your Pelican Platform
+                        </Typography>
+                    </Box>
                 </Box>
-                <Box pt={2} mx={"auto"}>
-                    { enabledServers !== undefined && enabledServers.includes("registry") &&
+                <Box mx={"auto"}>
+                    { enabledServers && enabledServers.includes("registry") &&
                         <>
-                            <Typography textAlign={"center"} variant={"h6"} component={"h2"}>
-                                For Outside Administrators
-                            </Typography>
-                            <Box display={"flex"} justifyContent={"center"} my={2} mb={3}>
+                            <Box display={"flex"} justifyContent={"center"} mb={1}>
                                 <Button
                                     size={"large"}
                                     href={"/api/v1.0/auth/cilogon/login?next_url=/view/registry/"}
```

Comment:
```suggestion
                                    href={"/api/v1.0/auth/oauth/login?next_url=/"}
```

I updated the endpoint in #963 
Since Origin admin will also be able to login via OAuth, we should redirect user to `/` after they are authenticated, or even better, we should redirect them to where they were before hitting the login page